sops/read_data.go

```diff
@@ -15,7 +15,7 @@ import (
 func readData(content []byte, format string) (map[string]string, string, error) {
 	cleartext, err := decrypt.Data(content, format)
 	if userErr, ok := err.(sops.UserError); ok {
-		err = fmt.Errorf(userErr.UserError())
+		err = fmt.Errorf("%s", userErr.UserError())
```

Comment:
I think we'll just want this:
```suggestion
		err = userErr.UserError()
```

That way, if `UserError` returns a wrapped error, it is preserved rather than stringified.

Reply:
UserError() return string not Error, Maybe err = userErr?

Reply:
Whoops, yes, `err = userErr` :facepalm: Re returning it directly, we could, but then we'd duplicate the format string from the other return site.

Reply:
So, which approach should I use?

Reply:
Go for `err = userErr` for now, as that is the minimal change so we don't dilute this PR too much